library/testing/TestSDKUtils.cxx

```diff
@@ -116,5 +116,24 @@ int TestSDKUtils(int argc, char* argv[])
     "collapsePath: remove dotdot", f3d::utils::collapsePath("/folder/../file.ext") == "/file.ext");
   test("collapsePath: expand home", f3d::utils::collapsePath("~/folder/file.ext").is_absolute());
 
+  //
+
+  test("getEnv: standard", f3d::utils::getEnv("F3D_TEST_ENV_STANDARD") == "TestEnv");
+  test("getEnv: empty", f3d::utils::getEnv("F3D_TEST_ENV_EMPTY") == std::nullopt);
+  test("getEnv: extended", f3d::utils::getEnv("F3D_TEST_ENV_EXTENDED") == "TestËnv");
+  test("getEnv: utf8", f3d::utils::getEnv("F3D_TEST_ENV_UTF8") == "🤔");
```

Comment:
Not super fan of adding non-ascii characters to C++ source code.  
What about that?
```suggestion
  test("getEnv: extended", f3d::utils::getEnv("F3D_TEST_ENV_EXTENDED") == "Test\xEBnv");
  test("getEnv: utf8", f3d::utils::getEnv("F3D_TEST_ENV_UTF8") == "\xF0\x9F\xA4\x94");
```